mathml/elements/mrow.json

```diff
@@ -36,7 +36,7 @@
               "version_added": "6"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "6"
```

Comment:
Same here, should remain `false` and sub features should be `false`, too.
https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mrow#Browser_compatibility